tools/probers/bazelrbe/bazelrbe.go

```diff
@@ -20,6 +20,7 @@ import (
 var (
 	bazelBinary = flag.String("bazel_binary", "", "Path to bazel binary")
 	bazelArgs   = flag.String("bazel_args", "", "Space separated list of args to pass to Bazel")
+	proberName  = flag.String("prober_name", "", "Short, human-readable name of this prober")
```

Comment:
also worth noting that it should be a valid bazel package name

Reply:
Done